clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -371,8 +371,7 @@ private void doSend(ClientRequest clientRequest, boolean isInternalRequest, long
         } catch (UnsupportedVersionException e) {
             // If the version is not supported, skip sending the request over the wire.
             // Instead, simply add it to the local queue of aborted requests.
-            log.debug("Version mismatch when attempting to send {} to {}",
-                    clientRequest.toString(), clientRequest.destination(), e);
+            log.debug("Version mismatch when attempting to send {} to {}", builder, clientRequest.destination(), e);
```

Comment:
Hehe, I was debating that. Since other places didn't seem to be doing it, I did not do it here either. I pushed additional commits attempting to do this and improve how we output `node` information. Please take a look.